saleor/graphql/checkout/tests/benchmark/test_homepage.py

```diff
@@ -138,4 +149,20 @@ def test_user_checkout_details(user_api_client, customer_checkout, count_queries
           }
         }
     """
-    get_graphql_content(user_api_client.post_graphql(query))
+    cnt = get_graphql_content(user_api_client.post_graphql(query))
```

Comment:
There is no need to check the response here. We just want to record a number of queries

Reply:
OK, Removed